gitlab/__init__.py

```diff
@@ -780,6 +780,12 @@ def __init__(self, gl, url, query_data, get_next=True, **kwargs):
         self._query(url, query_data, **kwargs)
         self._get_next = get_next
 
+        # Preserve kwargs for subsequent queries
```

Comment:
Should this happen before https://github.com/python-gitlab/python-gitlab/pull/1124/commits/1d011ac72aeb18b5f31d10e42ffb49cf703c3e3a#diff-15f1286708cbbf354eafe7b7f49a1622L780? And at that line do:

```python
self._query(url, query_data, **self._kwargs)
```

That way this class would be internally consistent with how it calls `self._query()` in both `__init__()` and `next()`.

Reply:
Yeah, that was my thinking as well.